packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -439,13 +456,22 @@ class InteractiveGraphEditor extends React.Component<Props> {
                                     !!this.props.correct?.showSides
                                 }
                                 onChange={() => {
-                                    this.props.onChange({
-                                        correct: {
-                                            ...this.props.correct,
-                                            showSides:
-                                                !this.props.correct.showSides,
-                                        },
-                                    });
+                                    if (this.props.graph?.type === "polygon") {
+                                        this.props.onChange({
+                                            correct: {
+                                                ...this.props.correct,
+                                                showSides:
+                                                    !this.props.correct
+                                                        .showSides,
+                                            },
+                                            graph: {
+                                                ...this.props.graph,
+                                                showSides:
+                                                    !this.props.correct
```

Comment:
Question: I might have asked this when we were pairing, but, should this be from `graph` instead of `correct`?

Reply:
Yes! Graph and correct are largely the same, but we don't want `coords` to come from `correct`. That would result in the right side preview changing to the correct answer whenever the checkbox is checked. We want `coords` (and other props) to be the same as whenever `graph` already was.

Reply:
Yes! I think it doesn't matter that much since they should be the same, but it should be `graph` as a principle. And graph also has better type safety 👍🏽 I'll change that.

Reply:
I did it right for angles but not sides 🤦🏽‍♀️